### PR TITLE
HOTT-956: Expose introductory notes

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

[HOTT-956](https://transformuk.atlassian.net/browse/HOTT-956)

### What?

I have added/removed/altered:

- [x] Added api support for introductory_notes field on schemes
- [x] Include the introductory_notes in the expanding explanation field below the rules on the Rules of Origin tab
- [x] Don't show the expanding notes if there are no rules for the chosen country

### Why?

I am doing this because:

- We want to show the introductory notes information on the Rules of Origin tab

### CODE REVIEW

File: app/views/rules_of_origin/_introductory_notes.html.erb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <p>
+      If the heading in column 1 starts with ‘ex’, the rules in column 3
+      <!--(and 4 if present)//--> will only apply to the product as described
+      in column 2.
+    </p>
+
+    <h2 class="govuk-heading-m">Introductory notes</h2>
+
+    <div class="tariff-markdown">
+      <%= govspeak introductory_notes.to_s %>
```
Comment (alexdesi) [inline app/views/rules_of_origin/_introductory_notes.html.erb]: do we need `to_s` here?

File: app/views/rules_of_origin/_tab.html.erb [hunk 2]
```diff
@@ -15,14 +15,15 @@
       <%= render 'rules_of_origin/rules_table',
                  country_name: country_name,
                  rules: rules %>
+
+      <%= render partial: 'rules_of_origin/introductory_notes',
+                 collection: rules_of_origin.map(&:introductory_notes) %>
```
Comment (alexdesi) [inline app/views/rules_of_origin/_tab.html.erb:20]: maybe adding `introductory_notes` as a public method to get them, instead of map here? 🤔

**Review by alexdesi — APPROVED** [review]
it looks good 👍 
just added 2 minor comments

File: app/views/rules_of_origin/_introductory_notes.html.erb (hunk 1 above)
Comment (jebw) [reply app/views/rules_of_origin/_introductory_notes.html.erb]: safety net incase its nil and govsoeak doesn't handle nil, possible overkill - will change it to `if present?`

File: app/views/rules_of_origin/_tab.html.erb (hunk 2 above)
Comment (jebw) [reply app/views/rules_of_origin/_tab.html.erb:20]: Problem is rules of origin is an `Array` of `RulesOfOrigin::Scheme` not a specific class, so not sure where I would add it?

**Pull request merged**